go.mod

```diff
@@ -26,25 +26,30 @@ require (
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/poy/kontext v0.0.0-20190322194304-59ced15e96b1
 	github.com/poy/service-catalog v0.0.0-20190305064623-db385b1d332c
-	github.com/russross/blackfriday v0.0.0-20151117072312-300106c228d5 // indirect
 	github.com/segmentio/textio v1.2.0
-	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
 	github.com/sirupsen/logrus v1.3.0 // indirect
 	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/pflag v1.0.3 // indirect
+	go.opencensus.io v0.22.0 // indirect
 	go.uber.org/atomic v1.3.2 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
-	go.uber.org/zap v1.9.1 // indirect
+	go.uber.org/zap v1.9.1
 	golang.org/x/crypto v0.0.0-20190426145343-a29dc8fdc734 // indirect
 	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
 	golang.org/x/sys v0.0.0-20190502175342-a43fa875dd82 // indirect
 	google.golang.org/appengine v1.5.0 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 	gotest.tools v2.2.0+incompatible // indirect
 	k8s.io/api v0.0.0-20181121191454-a61488babbd6
 	k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
 	k8s.io/client-go v2.0.0-alpha.0.0.20190226174127-78295b709ec6+incompatible
 	k8s.io/klog v0.3.0 // indirect
 	k8s.io/kubernetes v1.14.2
+
 )
+
+replace go.opencensus.io => go.opencensus.io v0.20.2
```

Comment:
Mind adding some comments for this replacements?
I can see the context for these becoming lost in history.
If there's eventual work to remove these, then should we open an issue and add that to the comment as well?